tests/format/flow/as-satisfies-expression/jsfmt.spec.js

```diff
@@ -0,0 +1 @@
+run_spec(import.meta, ["flow"], { errors: { "babel-flow": true } });
```

Comment:
If tests are both valid in TS and Flow  we can  add `typescript` parser here, so we can make them print consistently if possible.  If code not valid in TS, please ignore.

Reply:
The tests in general are not valid for both TS and Flow. We still have a variety of syntactic differences (though over time, it's getting a bit closer as you see).